packages/p2p/src/components/order-details/order-details.scss

```
@@ -175,7 +181,7 @@ $card-width: 456px;
         @include mobile {
             flex: 1;
             min-height: 0;
-            padding: 1.6rem;
+            padding: 1.6rem 0rem;
```

You can just give the value as 0 without the unit. 

```suggestion
            padding: 1.6rem 0;
```